scripts/openvino_accelerate.py

```diff
@@ -673,6 +673,27 @@ def callback(iter, t, latents):
 
     devices.torch_gc()
 
+    # TODO: error for len(output_images) > 1
+    if override_hires:
+        if upscaler == "Latent":
+            model_state.recompile = 1
+            shared.sd_diffusers_model = get_diffusers_sd_model(model_config, sampler_name, 1)
+            shared.sd_diffusers_model.scheduler = set_scheduler(shared.sd_diffusers_model, sampler_name)
+            for i in range(len(output_images)):
+                p.batch_index = i
+                output_images[i]=output_images[i].resize((int(output_images[i].width * upscale_by), int(output_images[i].height * upscale_by)))
+                output_images[i] = shared.sd_diffusers_model(
+                            prompt=p.prompts,
+                            negative_prompt=p.negative_prompts,
+                            num_inference_steps=hires_steps,
+                            guidance_scale=p.cfg_scale,
+                            generator=generator,
+                            callback = callback,
+                            callback_steps = 1,
+                            image=output_images[i],
+                            strength=d_strength
+                ).images[0]
+
     res = Processed(

```

Comment:
@likholat , the PR looks good to me, but want to check if you tried diffusers latent upscale pipeline and if the result is consistent with it when you use txt2img and img2img back to back. It is great that you are efficiently reusing the components. I tried on a 16 GB system. Upscale factor of 1.5 worked for me, but when I tried 2, it ran out of memory. @qiacheng is also going to try it out on GPUs. If no major issues, we can merge.